packages/wallets/component-tests/menu.spec.tsx

```diff
@@ -26,6 +26,7 @@ test.describe('Wallets - Traders Hub', () => {
         });
         await page.goto(`${baseURL}/wallets`);
 
+        // eslint-disable-next-line testing-library/prefer-screen-queries
```

Comment:
There must be a way to configure the `integration-tests` folders (or component tests when renamed) so eslint knows it's using playwright.